client/my-sites/site-settings/publishing-tools/post-by-email.tsx

```diff
@@ -0,0 +1,168 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { ToggleControl } from '@wordpress/components';
+import clsx from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import ClipboardButtonInput from 'calypso/components/clipboard-button-input';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import SupportInfo from 'calypso/components/support-info';
+import JetpackModuleToggle from 'calypso/my-sites/site-settings/jetpack-module-toggle';
+import { useGetPostByEmail } from 'calypso/my-sites/site-settings/publishing-tools/hooks/use-get-post-by-email';
+import { useRegeneratePostByEmailMutation } from 'calypso/my-sites/site-settings/publishing-tools/hooks/use-regenerate-post-by-email-mutation';
+import { useTogglePostByEmailMutation } from 'calypso/my-sites/site-settings/publishing-tools/hooks/use-toggle-post-by-email';
+import { useDispatch, useSelector } from 'calypso/state';
+import { regeneratePostByEmail as jetpackRegeneratePostByEmail } from 'calypso/state/jetpack/settings/actions';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+import isJetpackModuleActive from 'calypso/state/selectors/is-jetpack-module-active';
+import isJetpackModuleUnavailableInDevelopmentMode from 'calypso/state/selectors/is-jetpack-module-unavailable-in-development-mode';
+import isJetpackSiteInDevelopmentMode from 'calypso/state/selectors/is-jetpack-site-in-development-mode';
+import isRegeneratingJetpackPostByEmail from 'calypso/state/selectors/is-regenerating-jetpack-post-by-email';
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+type PostByEmailSettingProps = {
+	isFormPending: boolean;
+	address?: string;
+};
+
+const noticeConfig = {
+	duration: 4000,
+};
+
+const moduleSlug = 'post-by-email';
+
+export const PostByEmailSetting = ( { isFormPending, address }: PostByEmailSettingProps ) => {
+	const selectedSiteId = useSelector( getSelectedSiteId ) || 0;
+	const siteIsJetpack = useSelector( ( state ) => isJetpackSite( state, selectedSiteId ) );
+	const siteIsAtomic = useSelector( ( state ) => isSiteAutomatedTransfer( state, selectedSiteId ) );
+	const jetpackRegeneratingPostByEmail = useSelector( ( state ) =>
+		isRegeneratingJetpackPostByEmail( state, selectedSiteId )
+	);
+	const jetpackPostByEmailIsActive = !! useSelector( ( state ) =>
+		isJetpackModuleActive( state, selectedSiteId, moduleSlug )
+	);
+	const siteInDevMode = useSelector( ( state ) =>
+		isJetpackSiteInDevelopmentMode( state, selectedSiteId )
+	);
+	const moduleUnavailableInDevMode = useSelector( ( state ) =>
+		isJetpackModuleUnavailableInDevelopmentMode( state, selectedSiteId, moduleSlug )
+	);
+	const moduleUnavailable = siteInDevMode && moduleUnavailableInDevMode;
+	const { data: simpleSitePostByEmailSettings } = useGetPostByEmail( selectedSiteId );
```

Comment:
```suggestion
	const moduleUnavailable = siteInDevMode && moduleUnavailableInDevMode;

	const { data: simpleSitePostByEmailSettings } = useGetPostByEmail( selectedSiteId );
```
Nit, but a newline here would be nice to delineate between Jetpack and Simple stuff.